Development Progress - week 01

rayapetkova opened the pull request:
Merging changes from `Experimental` to `Development`

rayapetkova committed:
Update subcategories menu

- Added all photos
- Made one common function in pyqt5-gui - main_menu_dev

karastoyanov committed:
Merge pull request #41 from Stake-And-Rope/raya_development

Update subcategories menu

karastoyanov committed:
major main_menu fixes

 - changes from main_menu_dev transfered to main_menu file (@rayapetkova ac3d5f8)
 - new subcategories window call with one single function
 - top layout buttons size changed (fixed heigth)
 - relantionship between files fixed (main > login/register > main_menu > subcategories)
 - code cleaning/refactoring

Co-Authored-By: Raya Petkova <122923696+rayapetkova@users.noreply.github.com>

karastoyanov committed:
Main Menu Update

karastoyanov committed:
Print Function Updated

karastoyanov committed:
Merge pull request #44 from Stake-And-Rope/raya_development

Merge `raya_development` with `experimental`

karastoyanov committed:
Merge pull request #45 from Stake-And-Rope/alex_development

Merge `alex_development` into `experimental`

karastoyanov reviewed (approved):
Changes from previous personal branches are approved. 

:warning: Check the [last meeting](https://app.eraser.io/workspace/1ZEtbIuCPdTmk2hVUNVJ) notes and proceed working on `experimental` branch

karastoyanov committed:
Update Meeting Notes_week01

karastoyanov committed:
Main Menu major update
 - update user settings groupbox/layout added
 - on call, categories groupbox is being removed and user settings
   replaces it

karastoyanov committed:
user settings update

karastoyanov commented on pyqt5-gui/main_menu.py:
```diff
@@ -243,42 +243,85 @@ def open_func(subcat_name):
         """OPEN EDIT ACCOUNT LAYOUT/REPLACE CATEGORIES LAYOUT"""
```
Fix phone number label and textbox appearance in the for loop

karastoyanov replied:
Fixed with https://github.com/Stake-And-Rope/market-app/commit/c937fae0106bc61c90d87397c5e630778f580662

Details about the fix are described in #42

karastoyanov reviewed (changes requested):
update user settings code review

* since last commit ffa0991 phone number doesn't appear in the layout. Modify this block of code:
```python
            text_labels = deque(['First Name', 'Last Name', 'Current Email Address', 'Phone Number'])
            text_labels_length = len(text_labels)
            for i in range(text_labels_length - 1):
                if i != 3:
                    current_text_label = QLabel()
                    current_text_label.setText(text_labels.popleft())
                    current_text_label.setFont(QFont(fonts[0], 12))

                    current_line_edit = QLineEdit()
                    current_line_edit.setText(result[i])
                    current_line_edit.setFont(QFont(fonts[0], 12))
                    current_line_edit.setMaximumWidth(250)
                else:
                    current_text_label = QLabel()
                    current_text_label.setText("New Email Address")
                    current_text_label.setFont(QFont(fonts[0], 12))

                    current_line_edit = QLineEdit()
                    current_line_edit.setFont(QFont(fonts[0], 12))
                    current_line_edit.setMaximumWidth(250)
```

karastoyanov committed:
show/hide different groupbox on gridlayout(1,1)

karastoyanov committed:
update user settings menu fix

karastoyanov committed:
Update user data - not fully implemented

karastoyanov committed:
Fix update_user function preventing re-opening the categoroies

rayapetkova committed:
Added images and made a few changes in the subcategories_menu

- Added 3 subcategories for the category Food - Meat, Fruits, Vegetables (forgot to do it the last time)
- Added a new image for a background to the subcategories menu
